src/commands/game_commands/lookup.ts

```diff
@@ -78,263 +362,51 @@ export default class LookupCommand implements BaseCommand {
-            views = kmqSongEntry.views;
-            publishDate = kmqSongEntry.publishDate;
-
-            const durationInSeconds = (
-                await dbContext
-                    .kmq("cached_song_duration")
-                    .where("vlink", videoID)
-                    .first()
-            )?.duration;
-
-            // duration in minutes and seconds
-            if (durationInSeconds) {
-                const minutes = Math.floor(durationInSeconds / 60);
-                const seconds = durationInSeconds % 60;
-                songDuration = `${minutes}:${
-                    seconds < 10 ? "0" : ""
-                }${seconds}`;
-            }
 
-            const session = Session.getSession(guildID);
-            includedInOptions = [
-                ...(
-                    await SongSelector.getFilteredSongList(
-                        await GuildPreference.getGuildPreference(guildID),
-                        await isPremiumRequest(session, message.author.id)
-                    )
-                ).songs,
-            ]
-                .map((x) => x.youtubeLink)
-                .includes(videoID);
-
-            logger.info(
-                `${getDebugLogHeader(
-                    message
-                )} | KMQ song lookup. videoID = ${videoID}. Included in options = ${includedInOptions}.`
-            );
-        } else {
-            description = LocalizationManager.localizer.translate(
-                guildID,
-                "command.lookup.notInKMQ",
-                { link: daisukiLink }
-            );
-
-            const isKorean = locale === LocaleType.KO;
-            songName =
-                daisukiSongEntry.kname && isKorean
-                    ? daisukiSongEntry.kname
-                    : daisukiSongEntry.name;
-
-            const artistNameQuery = await dbContext
-                .kpopVideos("app_kpop_group")
-                .select("name", "kname")
-                .where("id", daisukiSongEntry.id_artist)
-                .first();
-
-            artistName =
-                artistNameQuery.kname && isKorean
-                    ? artistNameQuery.kname
-                    : artistNameQuery.name;
-
-            songAliases = [...daisukiSongEntry.name_aka.split(";")].join(", ");
-            songAliases += songAliases
-                ? `, ${daisukiSongEntry.kname}`
-                : daisukiSongEntry.kname;
-
-            artistAliases =
-                State.aliases.artist[artistNameQuery.name]?.join(", ");
-
-            views = daisukiSongEntry.views;
-            publishDate = new Date(daisukiSongEntry.publishedon);
-
-            logger.info(
-                `${getDebugLogHeader(
-                    message
-                )} | Non-KMQ song lookup. videoID = ${videoID}.`
-            );
-        }
+        if (isValidURL(arg)) {
+            let videoID: string = null;
+            if (arg.startsWith("<") && arg.endsWith(">")) {
```

Comment:
Is it still considered a valid url if it starts with < and ends with > ? Might need to move this before the if statement